content/_components/shopify-admin-v2/index.md

```diff
@@ -305,24 +308,23 @@ Lookup a set of objects by defined criteria list. Can be emitted in different wa
 #### Configuration Fields
 
 * **Object Type** - (dropdown, required): Object-type to lookup on. E.g `Customers`.
-* **Select basic fields for resulting object** - (dropdown, optional): Here provided only basic fields that can be included in resulting object, it may affect on query cost
-* **You can provide additional fields here** - (string, optional): Resulting object can be expanded using GraphQL request, it may affect on query cost
-
-<details close markdown="block"><summary><strong>Example for customer</strong></summary>
-
-  ```Graphql
-    addresses {
-      address1
-      country
-  }
-  ```
-
-</details>
-
+* **Select basic fields for resulting object** - (dropdown, optional): Here provided only basic fields that can be included in resulting object, it may affect on query cost.
```

Comment:
* **Select basic fields for resulting object** - (dropdown, optional): Here only basic fields can be included in the resulting object, it may affect query cost.